.github/workflows/ci-backend.yml

```diff
@@ -211,6 +211,10 @@ jobs:
 
             - name: Run migrations up to master
               run: |
+                  # We need to ensure we have requirements for the master branch
+                  # now also, so we can run migrations up to master.
+                  python -m pip install -r requirements-dev.txt
```

Comment:
A bit confused by this change, since the previous step was installing deps already?

Reply:
We check out master, then run migrate. If there are requirements removed as in this case, the master run will have missing imports.